WORKSPACE

```diff
@@ -50,6 +50,7 @@ http_file(
     sha256 = "f9c70f8f45af8623f85c06e2a84b0c37c5ec30764d1bccc381d43b01436dce1c",
     urls = [
         "https://copr-be.cloud.fedoraproject.org/results/%40virtmaint-sig/for-kubevirt/fedora-30-x86_64/00967888-libvirt/libvirt-libs-5.0.0-1.fc30.x86_64.rpm",
+	"https://storage.googleapis.com/builddeps/libvirt-libs-5.0.0-1.fc30.x86_64.rpm",
```

Comment:
Please upload the files with a filename equal to the shasum as a general pattern. I want to avoid scenarios where we have to invend filenames because the name is already taken.

Reply:
done